packages/vaadin-combo-box/test/dynamic-size.test.js

```diff
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync } from '@vaadin/testing-helpers';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import { flushComboBox, getViewportItems, scrollToIndex } from './helpers.js';
-import '../src/vaadin-combo-box.js';
+import '../vaadin-combo-box.js';
```

Comment:
Should we change this test to use `not-animated-styles.js` to avoid timing issues?

Reply:
Done